x-pack/plugins/observability_solution/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -172,21 +172,32 @@ export class ObservabilityAIAssistantClient {
           kibanaPublicUrl,
           simulateFunctionCalling,
           isPublic = false,
+          instructions: requestInstructions = [],
         } = params;
 
         const isConversationUpdate = persist && !!params.conversationId;
         const conversationId = persist ? params.conversationId || v4() : '';
         const title = params.title || '';
         const responseLanguage = params.responseLanguage || 'English';
-        const requestInstructions = params.instructions || [];
+
+        const registeredInstructions = functionClient.getInstructions();
+
+        const knowledgeBaseInstructions: UserInstruction[] = [];
+
+        if (responseLanguage) {
+          requestInstructions.push(
+            `You MUST respond in the users preferred language which is: ${responseLanguage}.`
+          );
+        }
+
+        let storedSystemMessage: string = ''; // will be set as soon as kb instructions are loaded
```

Comment:
I'm not super happy about this, but let's tackle it in https://github.com/elastic/kibana/issues/180633.

Reply:
I will open a PR after this that fixes it [`d005850` (#181255)](https://github.com/elastic/kibana/pull/181255/commits/d0058503bec0032665d58716eaf82f6957c65382)